client/src/components/dashboard/DashboardEditor.jsx

```diff
@@ -132,10 +144,13 @@ export default class DashboardEditor extends Component {
   handleEntityToggle(item, itemType) {
     const newEntities = Object.assign({}, this.props.dashboard.entities);
     const newLayout = this.props.dashboard.layout.slice(0);
+    const itemExists = this.props.dashboard.entities[item.id];
```

Comment:
what do you think if we use `entity` instead `itemExists` :) ?

Reply:
done